agreements/financial.md

```diff
@@ -50,10 +50,12 @@ Money spent by the Foundation falls under the following categories.
 
 #### Fixed Costs
 
-The board is responsible for managing the fixed costs from core funds, which are expected to be kept at ‘minimum viable’ to ensure smooth operations of core processes for the network. 
+The Board is responsible for managing the fixed costs from core funds, which are expected to be kept at ‘minimum viable’ to ensure smooth operations of core processes for the network. 
 
 At its discretion the board can add/remove a Financially Enabled Working Group to the Fixed Costs table. When it does so the Board takes on the responsibility of holding the working group to account for funds spent.
 
+At least once a year the Board will run a community engagement process to recieve insight/feedback on what the sponsors and communities priority is for strategic spend by the board.
```

Comment:
@joshuavial something like this.

The other thread i was thinking of writing into this was about reporting - but actually I think that is already covered by the current reporting cadence/requirement wherever that is.

Reply:
One thought I had was asking that the board participate in one of the assembly sprints per year. But maybe thats to complex.